main.nf

```diff
@@ -641,9 +650,18 @@ process featureCounts {
     file "${bam_featurecounts.baseName}_biotype_counts.txt" into featureCounts_biotype
 
     script:
+    }if (params.reverse_stranded){
```

Comment:
Typo - `}` here.

I would probably write this as follows:
```groovy
def featureCounts_direction = 0
if (params.reverse_stranded){
    featureCounts_direction = 2
} else if (params.forward_stranded) {
    featureCounts_direction = 1
}
```
Rather than defining it three times.